spec/core/integration/EnvSpec.js

```diff
@@ -1894,6 +1894,44 @@ describe("Env integration", function() {
     env.execute();
   });
 
+  it('reports test properties', function(done) {
+      var env = new jasmineUnderTest.Env(),
+        reporter = jasmine.createSpyObj('reporter', ['jasmineDone', 'suiteDone', 'specDone']);
+
+      reporter.specDone.and.callFake(function(e) {
+        expect(e.properties).toEqual({a: 'Bee'});
+        done();
+      });
+
+      env.addReporter(reporter);
+      env.it('calls setProperty', function() {
+        env.setProperty('a', 'Bee')
+      });
+      env.execute();
+  });
+
+  it('throws an exception if you try to setProperty outside of a runnable', function (done) {
+    var env = new jasmineUnderTest.Env(),
+       exception;
+
+    env.describe("a suite", function () {
+      try {
+        env.setProperty('a prop', 'val');
```

Comment:
This reads as if we're setting a property on env, not on the current spec. I think the `setProperty` method probably has to be on `env`, but can we come up with a better name for it? Maybe something like `setSpecProperty`.

Reply:
done